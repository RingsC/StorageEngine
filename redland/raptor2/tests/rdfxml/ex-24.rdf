<?xml version="1.0"?>
<rdf:RDF xmlns:ex="http://example.org/foo#"
         xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">

  <!-- Check parseType daml:Collection with a typed node inside -->

  <rdf:Description rdf:about="http://example.org/node1">
    <ex:prop1 rdf:parseType="daml:collection">
      <ex:node2>
        <ex:prop2 rdf:resource="http://example.org/node3" />
      </ex:node2>
    </ex:prop1>
  </rdf:Description>

</rdf:RDF>
