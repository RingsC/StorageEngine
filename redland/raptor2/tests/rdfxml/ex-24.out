_:genid2 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.daml.org/2001/03/daml+oil#List> .
_:genid2 <http://www.daml.org/2001/03/daml+oil#first> _:genid1 .
_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/foo#node2> .
_:genid1 <http://example.org/foo#prop2> <http://example.org/node3> .
_:genid2 <http://www.daml.org/2001/03/daml+oil#rest> <http://www.daml.org/2001/03/daml+oil#nil> .
<http://example.org/node1> <http://example.org/foo#prop1> _:genid2 .
